cap/01-categorie.tex

```diff
@@ -390,6 +405,7 @@ \section{Esempi di categorie nella pratica matematica}\label{sec_esempi_cats}
 \index{Categoria!--- delle matrici}
 \begin{example}[Categoria delle matrici in \(\bbF\)]\label{ex_cat_matrici}
 	La categoria \(\bbF\emdash\cate{Mat}\) delle \emph{matrici a ingressi in \(\bbF\)} ha
+	\todo{Si dice `ingressi' o `entrate' in italiano?}
```

Comment:
buona domanda: ho visto usare entrambi (forse dovremmo controllare in un libro "standard" di Algebra Lineare, per es. la traduzione italiana del Lang)

Reply:
Ho controllato, Cailotto https://www.math.unipd.it/~maurizio/m2m/AGLQ910pp.pdf usa "entrate" e tanto mi basta; Lang usa "componente" o "termine $i,j$-esimo", ma entrata mi piace di più.